### DISCUSSION TITLE
how to integrate seq into java springboot project more easily

### DISCUSSION THREAD

**Discussion by levonyang** [datalust/seq-tickets#1602] [Help]
I once integrated seq into a .net core project. It was very easy to be done and the functionality meets the requirements. Now I am developing a springboot project and I also try to use this powerful logging tool into the project, but it is not very easy to do.

**Comment by liammclennan** [comment]
Hi @levonyang 

Probably the easiest way to log to Seq from a springboot project is to use [serilogj](https://docs.datalust.co/docs/using-java). 

```java
Log.setLogger(new LoggerConfiguration()
    .writeTo(seq("http://localhost:5341"))
    .createLogger());

Log.information("Hello from {lang}!", "Java");
```

Have you tried serilogj?
